login.css

```diff
@@ -0,0 +1,40 @@
+body{
+    background-color: #eeeeee;
+    height:90vh;
+    display:flex;
+    align-items:center;
+    justify-content:center;
+}
+
+.form{
+    background-color: white;
+    border-radius: 1rem;
+    box-shadow:0 0 0.2rem 0rem;
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    padding:1rem;
+    gap:1rem;
+}
+
+.form-header{
+    color:var(--primary-color);
+    margin:0;
```

Comment:
Suggestion: Is this needed??i think it is 0 by default so you can remove this...
One thing you can do is reset margin at the beginning of styling...